src/test/java/com/yahoo/bullet/bql/integration/ExpressionTest.java

```diff
@@ -0,0 +1,453 @@
+        Assert.assertEquals(query.getProjection().getFields().get(4), new Field("a = 5", binary(field("a", Type.LONG),
+                                                                                                value(5),
+                                                                                                Operation.EQUALS,
+                                                                                                Type.BOOLEAN)));
+        Assert.assertEquals(query.getProjection().getFields().get(5), new Field("a != 5", binary(field("a", Type.LONG),
+                                                                                                 value(5),
+                                                                                                 Operation.NOT_EQUALS,
+                                                                                                 Type.BOOLEAN)));
+        Assert.assertEquals(query.getProjection().getFields().get(6), new Field("a > 5", binary(field("a", Type.LONG),
+                                                                                                value(5),
+                                                                                                Operation.GREATER_THAN,
+                                                                                                Type.BOOLEAN)));
+        Assert.assertEquals(query.getProjection().getFields().get(7), new Field("a < 5", binary(field("a", Type.LONG),
+                                                                                                value(5),
+                                                                                                Operation.LESS_THAN,
+                                                                                                Type.BOOLEAN)));
+        Assert.assertEquals(query.getProjection().getFields().get(8), new Field("a >= 5", binary(field("a", Type.LONG),
+                                                                                                 value(5),
+                                                                                                 Operation.GREATER_THAN_OR_EQUALS,
+                                                                                                 Type.BOOLEAN)));
+        Assert.assertEquals(query.getProjection().getFields().get(9), new Field("a <= 5", binary(field("a", Type.LONG),
+                                                                                                 value(5),
+                                                                                                 Operation.LESS_THAN_OR_EQUALS,
+                                                                                                 Type.BOOLEAN)));
+    }
+
+    @Test
+    public void testBinaryOperationsAnyAll() {
+        build("SELECT a = ANY [5], a = ALL [5], a != ANY [5], a != ALL [5], a > ANY [5], a > ALL [5], a < ANY [5], " +
```

Comment:
Can we add multiple entries to these anys and alls rather than just 5?

Reply:
I can, but there's no reason to. it's not like anything is being evaluated. i just need an integer list there